src/device_trezor/README.md

```diff
@@ -15,19 +15,29 @@ Please, refer to [monero readme](https://github.com/trezor/trezor-firmware/blob/
 
 ## Dependencies
 
-Trezor uses [Protobuf](https://protobuf.dev/) library. As Monero is compiled with C++14, the newest Protobuf library version cannot be compiled because it requires C++17 (through its dependency Abseil library).
-This can result in a compilation failure.
+Trezor uses [Protobuf](https://protobuf.dev/) library. Monero is now compiled with C++ 17 by default.
+Protobuf v21+ is required. Note that Protobuf v23+ requires C++ 17.
```

Comment:
Why is v21 required?

Reply:
v21 (and greater) is the tested version. Would you prefer different wording?

Reply:
Yes, maybe reword it as something lime v21+ is tested, older versions are not guaranteed to work.

Reply:
done :)